specs/UI_import_gluster_cluster_workflow.adoc

```diff
@@ -0,0 +1,218 @@
+:imagesdir: ./images/
+
+= Import gluster cluster workflow
+
+== Problem description
+
+This specification describes import gluster cluster workflow which is a mechanism to add an gluster storage cluster into tendrl for management.
+
+View design will look like:
+
+image::import.png[Import workflow]
+
+== Use Cases
+
+This addresses the use case of importing existing gluster clusters into tendrl for management.
+
+The workflow is as follows:
+
+* API will provide the list of clusters(imported and ready to import) and those clusters will be displayed on cluster list page.
+
+* A button will displayed for each cluster which are not imported and are ready to import.
+
+* User will click on the import button for the cluster, he wants to import.
+
+* User will be redirected to new view.
+
+* UI will filter the available nodes for importing the cluster.
```

Comment:
@anivargi Can you please provide the API which will provide the nodes detail along with their response.